_posts/2020-03-28-Server-Rpi.md

```diff
@@ -11,7 +11,7 @@ This guide shows you how to install Jamulus on a Raspberry Pi.
 
 ## Server Installation - Raspberry Pi
 
-Since there are no official armhf .deb files yet, you'll need to compile Jamulus for your Pi from source.
+Although there are official armhf .deb files ([GUI download]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) [headless download]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})) which are recommended to be used, you can also compile Jamulus for your Pi from source.
```

Comment:
How about something stronger?
```suggestion
We now recommend you use the official armhf .deb files ([GUI download]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) [headless download]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})).  However, you can still compile Jamulus for your Pi from source, if you wish.
```

Reply:
Ok. Done. You're correct. It's better to hint at the official files.